streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableAggregate.java

```diff
@@ -85,10 +85,12 @@ public void process(final K key, final Change<V> value) {
             final ValueAndTimestamp<T> oldAggAndTimestamp = store.get(key);
             final T oldAgg = getValueOrNull(oldAggAndTimestamp);
             final T intermediateAgg;
+            long newTimestamp = context().timestamp();
```

Comment:
nit: to be consistent and also easier reading for others, we could do the same pattern like above classes, i.e. not initialize the value, just assign within the if-else.